kuma/search/management/commands/reindex.py

```diff
@@ -23,4 +23,6 @@ def handle(self, *args, **options):
         if not 1 <= percent <= 100:
             raise CommandError('percent should be between 1 and 100')
 
-        WikiDocumentType.reindex_all(options['chunk_size'], percent=percent)
+        message = WikiDocumentType.reindex_all(options['chunk_size'],
+                                               percent=percent)
+        print unicode(message)
```

Comment:
`sys.stdout.write` perhaps? I think I've seen that used in django management commands. Any reason why to not use print?